source/connection-monitoring-and-pooling/tests/pool-clear-schedule-run-interruptInUseConnections-false.yml

```diff
@@ -1,6 +1,6 @@
 version: 1
 style: unit
-description: Pool clear SHOULD schedule the next background thread run immediately (interruptInUseConnections: false)
+description: Pool clear SHOULD schedule the next background thread run immediately (interruptInUseConnections = false)
```

Comment:
Assuming the colon was a problem in the original version, quoting the entire value would have also worked here.